src/hooks/__tests__/Recoil_useRecoilCallback-test.js

```diff
@@ -149,11 +149,13 @@ describe('useRecoilCallback', () => {
     // Something with React and Jest requires this extra kick...
     const [Kick, kick] = componentThatReadsAndWritesAtom(kickAtom);
 
+    /* eslint-disable react/jsx-key */
     const container = renderElements([
       <Component />,
       <ReadsAtom atom={anAtom} />,
       <Kick />,
     ]);
+    /* eslint-enable react/jsx-key */
```

Comment:
@Komalov i don't think we use that rule internally and it warns about unused suppression, can we disable it in here as well?

Reply:
Sure, I disabled this rule for public builds